10x/count/count.wdl

```diff
@@ -173,6 +182,17 @@ task chunk_reads_join {
     
     # Write out_chunks to their own file
     jq '.out_chunks' _outs > out_chunks.json
+
```

Comment:
... and also some explanation here.

Reply:
This is a very sad piece of code, but I added some comments. All I want to do is flatten a list of lists of files, but so far I seem to have to transfer everything to worker to do that.